src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/code/RuntimeStub.java

```diff
@@ -44,9 +44,7 @@ public void update(Observable o, Object data) {
 
   private static void initialize(TypeDataBase db) {
     Type type = db.lookupType("RuntimeStub");
-    callerMustGCArgumentsField                = type.getCIntegerField("_caller_must_gc_arguments");
-
-    // FIXME: add any needed fields
+    callerMustGCArgumentsField = type.getCIntegerField("_caller_must_gc_arguments");
```

Comment:
This field is in `CodeBlob` since JDK 23 [JDK-8329433](https://bugs.openjdk.org/browse/JDK-8329433) https://github.com/openjdk/jdk/blob/master/src/hotspot/share/code/codeBlob.hpp#L126

Reply:
Ah, right. I forgot you had mentioned that. I think that means I can rid of RuntimeStub and RuntimeBlob once this code is moved to CodeBlob.

Reply:
The tests are all passing.